build/configs/rtl8720e/Make.defs

```diff
@@ -120,7 +120,12 @@ ifeq ($(CONFIG_FRAME_POINTER),y)
   ARCHOPTIMIZATION += -fomit-frame-pointer -mapcs -mno-sched-prolog
 endif
 
-ARCHCFLAGS = -fno-builtin -mcpu=cortex-m33 -mfpu=fpv5-sp-d16 -fno-common
+ARCHCFLAGS = -fno-builtin -mcpu=cortex-m33 -mfpu=fpv5-sp-d16
```

Comment:
Please provide more information in the description of commit message. The current commit message is ambiguous and do not provide any information about  change.